tests/Tests.FeatureManagement/FeatureManagement.cs

```diff
@@ -483,5 +483,64 @@ public async Task SwallowsExceptionForMissingFeatureFilter()
 
             Assert.False(isEnabled);
         }
+
+        [Fact]
+        public async Task CustomSettingsProvider()
+        {
+            FeatureDefinition testFeature = new FeatureDefinition
+            {
+                Name = ConditionalFeature,
+                EnabledFor = new List<FeatureFilterConfiguration>()
+                {
+                    new FeatureFilterConfiguration
+                    {
+                        Name = "Test",
+                        Parameters = new ConfigurationBuilder().AddInMemoryCollection(new Dictionary<string, string>()
+                        {
+                           { "P1", "V1" },
+                        }).Build()
+                    }
+                }
+            };
+
+            var services = new ServiceCollection();
+
+            services
+                .Configure<FeatureManagementOptions>(options =>
+                {
+                    options.IgnoreMissingFeatureFilters = true;
```

Comment:
This does not seem needed.

Reply:
Good point, it doesn't. I have removed it.